### PR TITLE
Remove "crispy forms" integration

### PR DESCRIPTION
**Pull request by SchrodingersGat**
- No longer doing any back-end form rendering

### CODE REVIEW

**Comment by SchrodingersGat** [comment]
@matmair can we remove `./src/backend/InvenTree/InvenTree/forms.py` entirely now?

**Review by matmair — APPROVED** [review]

**Comment by matmair** [comment]
@SchrodingersGat we have a bunch of auth logic in there, I will have to take a good look

**Comment by SchrodingersGat** [comment]
@matmair ok I'll hold off on merging this one. It would be great if we can cleanly remove that whole file

**Comment by SchrodingersGat** [comment]
Yeah, let's move them to a different file. Do you want to do that in a separate PR after this one?

**Comment by matmair** [comment]
@SchrodingersGat I would appreciate it, not really a good feeling pushing to another persons branch

File: src/backend/InvenTree/InvenTree/settings.py [hunk 1]
```diff
@@ -1336,11 +1331,12 @@
     'signup': 'InvenTree.forms.CustomSignupForm',
     'add_email': 'allauth.account.forms.AddEmailForm',
     'change_password': 'allauth.account.forms.ChangePasswordForm',
-    'set_password': 'allauth.account.forms.SetPasswordForm',
+    # 'set_password': 'allauth.account.forms.SetPasswordForm',
```
Comment (matmair) [inline src/backend/InvenTree/InvenTree/settings.py:1334]: Q (blocking): why is this changed?

**Review by matmair — CHANGES_REQUESTED** [review]

**Comment by matmair** [comment]
I have created a PR for the moving out, to be merged after this

File: src/backend/InvenTree/InvenTree/settings.py (hunk 1 above)
Comment (SchrodingersGat) [reply src/backend/InvenTree/InvenTree/settings.py:1334]: Right, I commented this out to get the code to launch - can we remove the custom 'set_password' form in `settings.py`?

Comment (matmair) [reply src/backend/InvenTree/InvenTree/settings.py:1334]: I do not think so but I would have to try to be certain

**Comment by SchrodingersGat** [comment]
@matmair fixed that line, can you please approve so that I can merge?

**Review by matmair — APPROVED** [review]
